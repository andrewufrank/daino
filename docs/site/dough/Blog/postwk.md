---
title: postwk with image
author: AUF
date: Jan. 4, 2019
keywords: test
abstract: A silly text not needing an abstract.
pageTemplate: page3.yaml
---

Some text without meaning; really? with an image 

![first picture ]( /home/frank/Workspace8/ssg/docs/site/dough/resources/img/120-2026_IMG.JPG  "Logo Title Text 1")

absolute statt relative path "/resources/img/120-2026_IMG.JPG"
  An example post.  postwk but in blog - not in SubBlog
with some additional text 
which is always changed  

addition
