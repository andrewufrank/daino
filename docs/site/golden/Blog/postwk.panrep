Panrep {panyam = MetaPage {dyFn = "/home/frank/Workspace11/ssg/docs/site/dough/Blog/postwk.md", dyLink = "Blog/postwk.md", dyLang = DLenglish, dyTitle = "postwk with image", dyAbstract = "A silly text not needing an abstract.", dyAuthor = "AUF", dyDate = Just "Jan. 4, 2019", dyKeywords = "test", dyBibliography = Nothing, dyStyle = Nothing, dyReferences = Nothing, dyNoCite = Nothing, dyPublish = Nothing, dyIndexPage = False, dyIndexSort = Nothing, dyIndexEntry = IndexEntry {fn = "/home/frank/Workspace11/ssg/docs/site/dough/Blog/postwk.md", link = "Blog/postwk.md", title = "postwk with image", abstract = "A silly text not needing an abstract.", author = "AUF", date = "Jan. 4, 2019", publish = Nothing, indexPage = False, dirEntries = [], fileEntries = []}}, panpan = Pandoc (Meta {unMeta = fromList [("abstract",MetaInlines [Str "A",Space,Str "silly",Space,Str "text",Space,Str "not",Space,Str "needing",Space,Str "an",Space,Str "abstract."]),("author",MetaInlines [Str "AUF"]),("date",MetaInlines [Str "Jan.",Space,Str "4,",Space,Str "2019"]),("keywords",MetaInlines [Str "test"]),("pageTemplate",MetaInlines [Str "page3.yaml"]),("title",MetaInlines [Str "postwk",Space,Str "with",Space,Str "image"])]}) [Para [Str "Some",Space,Str "text",Space,Str "without",Space,Str "meaning;",Space,Str "really?",Space,Str "with",Space,Str "an",Space,Str "image"],Para [Image ("",[],[]) [Str "first",Space,Str "picture"] ("/home/frank/Workspace11/ssg/docs/site/dough/resources/img/120-2026_IMG.JPG","Logo Title Text 1")],Para [RawInline (Format "html") "<!-- ![first picture ]( ../resources/img/120-2026_IMG.JPG  \"Logo Title Text 1\")  -->"],Para [Str "absolute",Space,Str "statt",Space,Str "relative",Space,Str "path",Space,Str "\"/resources/img/120-2026_IMG.JPG\"",SoftBreak,Str "An",Space,Str "example",Space,Str "post.",Space,Str "postwk",Space,Str "but",Space,Str "in",Space,Str "blog",Space,Str "-",Space,Str "not",Space,Str "in",Space,Str "SubBlog",SoftBreak,Str "with",Space,Str "some",Space,Str "additional",Space,Str "text",SoftBreak,Str "which",Space,Str "is",Space,Str "always",Space,Str "changed"],Para [Str "addition"]]}