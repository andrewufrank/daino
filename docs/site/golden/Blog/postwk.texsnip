TexSnip {snipyam = MetaPage {dyFn = "/home/frank/Workspace11/ssg/docs/site/dough/Blog/postwk.md", dyLink = "Blog/postwk.md", dyLang = DLenglish, dyTitle = "postwk with image", dyAbstract = "A silly text not needing an abstract.", dyAuthor = "AUF", dyDate = Just "Jan. 4, 2019", dyKeywords = "test", dyBibliography = Nothing, dyStyle = Nothing, dyReferences = Nothing, dyNoCite = Nothing, dyPublish = Nothing, dyIndexPage = False, dyIndexSort = Nothing, dyIndexEntry = IndexEntry {fn = "/home/frank/Workspace11/ssg/docs/site/dough/Blog/postwk.md", link = "Blog/postwk.md", title = "postwk with image", abstract = "A silly text not needing an abstract.", author = "AUF", date = "Jan. 4, 2019", publish = Nothing, indexPage = False, dirEntries = [], fileEntries = []}}, unTexSnip = "Some text without meaning; really? with an image\n\n\\includegraphics{/home/frank/Workspace11/ssg/docs/site/dough/resources/img/120-2026_IMG.JPG}\n\nabsolute statt relative path \"/resources/img/120-2026\\_IMG.JPG\" An\nexample post. postwk but in blog - not in SubBlog with some additional\ntext which is always changed\n\naddition"}