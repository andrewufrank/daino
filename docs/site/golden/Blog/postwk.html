<!DOCTYPE html>
<!-- the master page for the pandoc templating mechanism
	needs page-title, page-title-postfix
		author date and keywords for indexing
    contentHtml, beforeContent, afterContent to wrap htmlcontent  
	uses style.css as default, other can be loaded with key css 
	-->
<html lang="DLenglish">
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />


  <style type="text/css">
      code{white-space: pre-wrap;}
      span.smallcaps{font-variant: small-caps;}
      span.underline{text-decoration: underline;}
      div.column{display: inline-block; vertical-align: top; width: 50%;}
	  </style>

	
	
          <meta name="author" content="AUF" />
              <meta name="dcterms.date" content="Jan. 4, 2019" />
              <meta name="keywords" content="test" />
    
	<title></title>
	
        <style>
            body {
            margin: 0;
            font-family: Arial, Helvetica, sans-serif;
            }

            .hero-image {
            background-image: url('');
            background-color: white;
            height: 200px;
            background-position: center;
            background-repeat: no-repeat;
            background-size: cover;
            position: relative;
            }

            .hero-text {
            transform: translate(-50%, -50%);
            color: white;
            text-shadow: 2px 2px 8px black;
            font-weight: bold;
            vertical-align: top;
            }
        </style>
    </head>

    <body>
    <!-- AF - my template for pandoc page (from the default) page3.dtpl -->
        

    <section class="hero">
        <div class="hero-image">
            
                <h1></h1>
                <h3></h3>
        <div/>
    </section>
    <section class="header">

            <section class="menu">
		<ul>
            			</ul>
    </section>
    <section>
                <header>
        <h1 class="title">postwk with image</h1>
        
                <p class="author">AUF</p>
                </header>
         
    </section>

    <section class=menu2>
		<ul>
            					<li>
				    <a href=Blog/postwk.html><em>postwk with image </em></a>
				    <br>A silly text not needing an abstract.		     
				    <span class=tiny>AUF "Jan. 4, 2019" 
            Blog/postwk.html Nothing
                    </span>
                    
    			</li>
            				</ul>
    </section>

        
<!-- blog text content section -->
    
    abstractA silly text not needing an abstract. 
    content-from-panrep-blocks<p>Some text without meaning; really? with an image</p>
<p><img src="/home/frank/Workspace11/ssg/docs/site/dough/resources/img/120-2026_IMG.JPG" title="Logo Title Text 1" alt="first picture" /></p>
<p><!-- ![first picture ]( ../resources/img/120-2026_IMG.JPG  "Logo Title Text 1")  --></p>
<p>absolute statt relative path "/resources/img/120-2026_IMG.JPG" An example post. postwk but in blog - not in SubBlog with some additional text which is always changed</p>
<p>addition</p>
    


        <p class=tiny>
        Produced with SGG  on  from  with master4.dtpl.
        </p>

            </body>


</html>
